### ISSUE TITLE
Extension configuration happening too late

### ISSUE THREAD

**Issue by pureza** [lkishalmi/gradle-gatling-plugin#104]
In my `build.gradle.kts` I've got the following:

```
configure<com.github.lkishalmi.gradle.gatling.GatlingPluginExtension> {
    includeTestOutput = false
}
```

However, it seems like in `GatlingPlugin::createConfiguration`, the check for `includeTestOutput` is happening before the snippet above runs, which results in `test` outputs being added to the classpath, which I'm trying to avoid.

Is this a known issue? Is it possible to somehow remove the test outputs from the `gatlingRuntime` dependencies after the extension has been applied?

I'm using gradle 5.6.4.

**Comment by eshepelyuk** [comment]
Could you reproduce the same with groovy config, no kotlin ?
Since we have test regarding this flag

**Comment by pureza** [comment]
Yes, the same issue happens with groovy config. 

Which test are you referring too?

**Comment by eshepelyuk** [comment]
Unit and functional tests from this repository.

**Comment by eshepelyuk** [comment]
@pureza fixed in [3.3.4](https://github.com/lkishalmi/gradle-gatling-plugin/releases/tag/3.3.4)
